README.md

```diff
@@ -1,73 +1,92 @@
-# Role Name
+# crypto_policies
 
-A template for an ansible role which configures some GNU/Linux subsystem or
-service. A brief description of the role goes here.
+This Ansible role manages system wide crypto policies.
```

Comment:
We use system-wide in man pages but someone from doc team should review.

Reply:
Thanks. Fix in all the places.